packages/fxa-settings/src/pages/ResetPassword/index.stories.tsx

```diff
@@ -39,7 +53,10 @@ function renderStory({ account, props, queryParams }: RenderStoryOptions = {}) {
 export const Default = () => renderStory();
 
 export const WithServiceName = () =>
-  renderStory({ queryParams: `service=${MozServices.MozillaVPN}` });
+  renderStory({
+    integrationType: IntegrationType.OAuth,
+    queryParams: `service=${MozServices.MozillaVPN}`,
```

Comment:
Random question, probably not for this PR. But do we still want to use these query params, or do we want to just mock the integraiton directly?

Reply:
Yeah good thinking, I definitely think it's nicer to just mock the integration. 👍 I started removing some query param logic in at least one story (I think it was a story still using UrlQueryParam data now that I think about it!) and it didn't render as expected, so I didn't spend a ton of time on it otherwise. My Storybook changes here do feel a bit rough around the edges, e.g. the correct "integration" mock isn't always used and right there's still some query param goodies around, but I didn't want to time sink here beyond getting the states rendering, especially because we are refactoring a lot of these reset password pages later. Not sure it's worth a follow up issue, we can address piecemeal as we refactor pages (or in a container component refactor ticket).